### ISSUE TITLE
Importing JSON via es6 modules

### ISSUE THREAD

**Issue by jackharrhy** [bluedrop-learning-networks/eslint-config-bluedrop#51]
The following code is currently invalid based on our current rules, since the extension is _not_ omitted, and can't be since it would then attempt to import `package.js` instead.

```js
import pkg from './package.json'; // tries to resolve ./package.json, but is currently error

import pkg from './package'; // will resolve ./package.js, not what we want!
```

So do we:

- Add an exception for the usage of `.json` (:green_heart: yes pls)
- Not import json using es6 modules (:red_circle: pls no)

**Comment by jackharrhy** [comment]
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md

**Comment by MitMaro** [comment]
Fixed in #54
